packages/i18n/src/locales/en/user.json

```diff
@@ -219,7 +219,8 @@
       "revoke": {
         "message": "Are you sure you want to revoke?"
       }
-    }
+    },
+    "thisApp": "This app"
```

Comment:
I think we can move this to app.json. It can be used in many places so. What do you say? @Nabin99